docs/root/intro/arch_overview/other_features/local_rate_limiting.rst

```diff
@@ -4,7 +4,8 @@ Local rate limiting
 ===================
 
 Envoy supports local (non-distributed) rate limiting of L4 connections via the
-:ref:`local rate limit filter <config_network_filters_local_rate_limit>`.
+:ref:`local rate limit listener filter <config_listener_filters_local_rate_limit>` and :ref:`local rate limit network filter <config_network_filters_local_rate_limit>`.
+The difference is `local rate limit listener filter <config_listener_filters_local_rate_limit>` processes the sockets before TLS handshake and filter matching.
```

Comment:
Wonder if we should write `and filter chain matching` to be more precise

Reply:
filter chain is better. thanks!